ui/package.json

```diff
@@ -135,7 +135,7 @@
     "@influxdata/clockface": "2.3.1",
     "@influxdata/flux": "^0.5.1",
     "@influxdata/flux-lsp-browser": "^0.5.11",
-    "@influxdata/giraffe": "0.23.0",
+    "@influxdata/giraffe": "^0.24.0",
```

Comment:
let's pull this part out into a separate PR. i know it's a little tedious, and that logically it's all about adding mosaic plot types, but i think from a risk perspective, and from a documenting-our-work-for-posterity perspective, it's better to have this be a separate thing. if something somehow breaks in this PR, it'll be hard to nail it down. if we add in 0.24.0 separately and it doesn't break anything, and then something breaks from this PR, it'll be clear what to fix. it'll also provide clear history in git - upgrading to giraffe 0.24.0 will look like a separate step followed by adding mosaic plot types behind a flag.